tools/deployment/test.py

```diff
@@ -140,4 +141,16 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    # main()
```

Comment:
this function should not be commented.

Reply:
forgot to uncomment it.......

Reply:
fixed in this pr https://github.com/open-mmlab/mmdetection/pull/7241